dist-material/release-docs/NOTICE

```diff
@@ -838,7 +838,43 @@ from and not be held liable to the user for any such damages as noted
 above as far as the program is concerned.
 
 ------
+===========================================================================
+nacos-1.3.0 Notice
+===========================================================================
+Nacos
+Copyright 2018-2019 The Apache Software Foundation
+
+This product includes software developed at
+The Alibaba MiddleWare Group.
+
+------
+This product has a bundle Spring Boot:
```

Comment:
We don't bundle with Spring Boot, maybe Nacos server is. But Notice base on what we used.

So basically, you could remove the NOTICE change.